### PR TITLE
[MINOR] Fix typo in CelebornConf

### PR DESCRIPTION
**Pull request by jiaoqingbo**
### What changes were proposed in this pull request?

Fix typo in CelebornConf

### Why are the changes needed?

Fix typo in CelebornConf

### Does this PR introduce _any_ user-facing change?

NO

### How was this patch tested?

Passing GA

### CODE REVIEW

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala [hunk 1]
```diff
@@ -3364,7 +3364,7 @@ object CelebornConf extends Logging {
       .createWithDefaultString("15s")
 
   val CLIENT_RESERVE_SLOTS_RACKAWARE_ENABLED: ConfigEntry[Boolean] =
-    buildConf("celeborn.client.reserveSlots.rackware.enabled")
+    buildConf("celeborn.client.reserveSlots.rackaware.enabled")
       .categories("client")
       .version("0.3.0")
```
Comment (waitinfuture) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:3369]: Could you update the version to 0.3.1?

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks! Merging to main/0.3

File: common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala [hunk 2]
```diff
@@ -3364,9 +3364,9 @@ object CelebornConf extends Logging {
       .createWithDefaultString("15s")
 
   val CLIENT_RESERVE_SLOTS_RACKAWARE_ENABLED: ConfigEntry[Boolean] =
-    buildConf("celeborn.client.reserveSlots.rackware.enabled")
+    buildConf("celeborn.client.reserveSlots.rackaware.enabled")
```
Comment (pan3793) [inline common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:3367]: this seems introduces a breaking change

Comment (waitinfuture) [reply common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:3367]: > 

Add an alternative?

Comment (pan3793) [reply common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala:3367]: Yes, I think so.

**Pull request closed without merging**